### PR TITLE
Change lexical scope power

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: src/js/tabs/power.js [hunk 1]
```diff
@@ -99,12 +99,12 @@ TABS.power.initialize = function (callback) {
             }
         }
 
-        var template = $('#tab-power-templates .voltage-configuration');
-        for (var index = 0; index < FC.VOLTAGE_METER_CONFIGS.length; index++) {
-            var destination = $('#voltage-meter-' + index + ' .configuration');
-            var element = template.clone();
+        template = $('#tab-power-templates .voltage-configuration');
```
Comment (McGiverGim) [inline src/js/tabs/power.js:102]: Reuse the template variable across the entire code maybe is not a good idea. I think is better and clearer to assign a different name for each of them, for example here `templateVoltageConfiguration`.
Caution with this change, not forget to rename where this template variable is used.
I think is only useful to reuse variable if the value is conditional

File: src/js/tabs/power.js [hunk 2]
```diff
@@ -211,19 +211,19 @@ TABS.power.initialize = function (callback) {
...
         $(element).find('.mah-drawn').attr('id', 'battery-mah-drawn');
         $(element).find('.amperage').attr('id', 'battery-amperage');
 
         destination.append(element.children());
 
-        var template = $('#tab-power-templates .battery-configuration');
-        var destination = $('.tab-power .battery .configuration');
-        var element = template.clone();
+        template = $('#tab-power-templates .battery-configuration');
+        destination = $('.tab-power .battery .configuration');
```
Comment (McGiverGim) [inline src/js/tabs/power.js:225]: The same for the `destination` variable that for the `template`, is better to have it's own variable at each use.

**Review by McGiverGim — CHANGES_REQUESTED** [review]
Only two things, I think they make a clearer code.

File: src/js/tabs/power.js (hunk 2 above)
Comment (haslinghuis) [reply src/js/tabs/power.js]: Did a rebase, much more clear now.

**Review by McGiverGim — APPROVED** [review]

**Review by mikeller — APPROVED** [review]

**Pull request merged**